src/parameterization_constants.jl

```diff
@@ -0,0 +1,16 @@
+"""
+Parameters for radiation parameterizations.
+"""
+@with_kw struct ParameterizationConstants{NF<:AbstractFloat}
+    fband::Matrix{Real}
+end    
+
+function ParameterizationConstants(P::Parameters)
+    @unpack NF, nband = P
+    fband = fill(NF(NaN), 400, nband)   # Energy fraction emitted in each longwave band = f(T)
```

Comment:
This can now be written as `fband = nans(NF,400,nband)` also the 400 looks weirdly hardcoded, any clue with 400 and not, say, 500?